_utility/send_gh_comment.sh

```diff
@@ -1,34 +1,43 @@
 #!/bin/bash
-set -e
+set +e
```

Comment:
Either create a usage() that displays the usage with an example or put that text in the header of this file as comments whichever is more appropriate - will help with the learning curve.